server/api/itemRouter.js

```diff
@@ -0,0 +1,115 @@
+const Item = require('../db/models/item');
+const Category = require('../db/models/category');
+const express = require('express');
+const router = express.Router();
+
+/**
+ * Add an item
+ * 
+ * req.body: {sourceLink: String,
+ *            highlight: String,
+ *            userId: mongoose.ObjectId,
+ *            placesId: id(optional),
+ *            comment: String (optional),}
+ * 
+ * response: {success: boolean,
+ *            message: String}
+ */
+router.post('/addItem', (req, res, next) => {
+  const newItem = req.body;
+  const item = new Item(newItem);
+  let response = {}; 
+  item.save((err) => {
```

Comment:
Unless there was a reason for this, can we enforce consistency between when to use await vs when to use a callback?

Reply:
+1

Reply:
Yes, fixed for all functions that still use callback.